AirLineAPI/Controllers/PassengersController.cs

```diff
@@ -5,30 +5,36 @@
 using Microsoft.AspNetCore.Mvc;
 using AirLineAPI.Db_Context;
 using AirLineAPI.Services;
-using Microsoft.EntityFrameworkCore;
 using AirLineAPI.Model;
 using Microsoft.AspNetCore.Http;
+using AutoMapper;
+using AirLineAPI.Dto;
 
 namespace AirLineAPI.Controllers
 {
     [Route("api/v1.0/[controller]")]
     [ApiController]
     public class PassengersController : ControllerBase
     {
-        private readonly IPassengerRepo repo;
-        public PassengersController(IPassengerRepo repo) => this.repo = repo;
-
+        private readonly IPassengerRepo _passengerRepo;
+        private readonly IMapper _mapper;
+       
+        public PassengersController(IPassengerRepo passengerRepo, IMapper mapper)
+        {
+            _passengerRepo = passengerRepo;
+            _mapper = mapper;
+        }
         //api/v1.0/Passengers
         [HttpGet]
         public async Task<ActionResult<Passenger[]>> GetPassenger([FromQuery] bool timeTable)
         {
             try
             {
-                if (repo == null)
+                if (_passengerRepo == null)
                 {
                     return NotFound();
```

Comment:
add comment example: return NotFound(Could not find passengers);

Reply:
tänker inte kommentera samma på alla här jag bara requestar changes

Reply:
det har blivit något fel med commiten för jag har ändrat men när jag commitar ändras det till backa det hände med alla mina error handling får fixa det efter rasten :)